Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```

Comment:
Non need for this file, see me previous comment.

Reply:
Done